integration/src/test/scala/org/knora/webapi/responders/v2/ValuesResponderV2Spec.scala

```diff
@@ -4324,7 +4325,8 @@ class ValuesResponderV2Spec extends CoreSpec with ImplicitSender {
     val deleteDate: Instant                       = Instant.now
     val deleteComment                             = Some("this value was incorrect")
 
-    val actual = UnsafeZioRun.run(
+    // TODO comment out
```

Comment:
I suppose that comment can be removed?

Reply:
Yup, the warning was obviously only about the `val actual`, not the code that generated the value for `actual`.